test/Sentry.Tests/SentryClientTests.cs

```diff
@@ -289,6 +290,85 @@ public void CaptureEvent_DisposedClient_ThrowsObjectDisposedException()
             _ = Assert.Throws<ObjectDisposedException>(() => sut.CaptureEvent(null));
         }
 
+        [Fact]
+        public void CaptureUserFeedback_Ignored_If_EventIdEmpty()
+        {
+            //Arrange
+            var sut = _fixture.GetSut();
+            var captureCalledEvent = new ManualResetEvent(false);
```

Comment:
Still using synchronizaiton here but no threading involved in this test. You fixed the other test, same situation here